src/Standards/Generic/Tests/CodeAnalysis/JumbledIncrementerUnitTest.1.inc

```diff
@@ -0,0 +1,83 @@
+<?php
+
+for ($i = 0; $i < 20; $i++) {
+    for ($j = 0; $j < 5; $i += 2) {
+        for ($k = 0; $k > 3; $i++) {
+            
+        }
+    }
+}
+
+for ($i = 0; $i < 20; $i++) {
+    for ($j = 0; $j < 5; $j += 2) {
+        for ($k = 0; $k > 3; $k++) {
+            
+        }
+    }
+}
+
+for ($i = 0; $i < 20; $i++) {
+    for ($j = 0; $j < 5; $j += 2) {
+        for ($k = 0; $k > 3; $j++) {
+            
+        }
+    }
+}
+
+for (; $i < 10; $i++) {
+    for ($j = 0;; $j++) {
+        if ($j > 5) {
+            break;
+        }
+        for (;; $k++) {
+            if ($k > 5) {
+                break;
+            }
+        }
+    }
+}
+
+for (; $i < 10; $i++) {
+    for ($j = 0;; $i++) {
+        if ($j > 5) {
+            break;
+        }
+        for (;; $i++) {
+            if ($k > 5) {
+                break;
+            }
+        }
+    }
+}
+
+for ($i = 0; $i < 20; $i++) :
+    for ($j = 0; $j < 5; $j += 2) :
+    endfor;
+endfor;
+
+for ($i = 0; $i < 20; $i++) :
+    for ($j = 0; $j < 5; $i += 2) :
+    endfor;
+endfor;
+
+// Sniff bails early when there is no incrementor in the outer for loop.
```

Comment:
```suggestion
// Sniff bails early when there is no incrementor in the third expression of the outer for loop.
```

Reply:
Looks like this fix-up was undone by the variable renaming commit ?

Reply:
Actually, I made this change on purpose but unfortunately, I failed to highlight that in the commit message. Sorry for the confusion.

When working on the variable renaming commit, I thought that the code comment would be more precise without referencing an incrementer, since the sniff is triggered for decrementer as well. However, I did not realize that I was changing a code comment that had been changed based on your suggestion in the previous commit.

That being said, since the PR is already merged, I think it is fine to keep the comment as is. It is just a detail.